### ISSUE TITLE
Add coinmarketcap and cryptocompare as exchange rate sources so we can support all new tokens

### ISSUE THREAD

**Issue by einsteinx2** [balancemymoney/balance-open#305]
Right now we use some major exchanges to get pricing information. Coinmarketcap, CryptoCompare, and other aggregators have APIs that allow access to their price ticker info just like exchanges, but they show ALL of the tokens, and all of the new ones. I had added Kucoin to get more new tokens, but adding one of the aggregators--or even better, multiple--would fill in the gaps better.

**Comment by einsteinx2** [comment]
Placing in 1.0.1, but these are really fast to do, so if I can find a few minutes I'll try and sneak it into 1.0. It's not just useful for the price tracker, it's essential that we have prices for all token types for the total balance in the Accounts view to show the correct amount.

**Comment by einsteinx2** [comment]
Actually I noticed during large fluctuations in bitcoin like the other day, Coinmarketcap is totally wrong on the price because it averages too many exchanges for bitcoin and doesn't update the price fast enough for many exchanges. I saw it as far as $2000 off the correct price.

CryptoCompare may be better, but I think in the near term, we should just add Kucoin and HitBTC to get new token prices as they launch.
